### ISSUE TITLE
How to add bearer token ?

### ISSUE THREAD

**Issue by garimak29** [cowinapi/developer.cowin#195]
I am trying to add the bearer token but its failing when i am trying to test the api on my browser directly

**Comment by iamabhishek-n** [comment]
@garimak29 You need to add the bearer to the request header, in a normal browser you can't change request headers, you can try using Rest clients like, Postman and add your Auth bearer and can perform the task.

**Comment by monodeepdas1215** [comment]
@Astro-Abhi Have you tried it with auth bearer tokens ?
I mean it is mentioned on their page to be eligible for auth token to the platform one have to be part of an organization with >=50 members.
Are you a solo dev who tried these apis with token ?
If so what is the procedure for a solo dev to get token for experimentation purposes ?

**Comment by iamabhishek-n** [comment]
@monodeepdas1215 you dont need bearer token to access public api`s

**Comment by anandvijay094** [comment]
> @monodeepdas1215 you dont need bearer token to access public api`s

we need bearer token to download  vaccine certificate from Public API .

**Comment by iamabhishek-n** [comment]
@anandvijay094 What are using currently Nodejs ? or any other ? and btw you cannot add token in normal browsers, you need to use scripts or API clients like postman

**Comment by anandvijay094** [comment]
> @anandvijay094 What are using currently Nodejs ? or any other ? and btw you cannot add token in normal browsers, you need to use scripts or API clients like postman

Yup i am using it in postman , but you can also use the bearer token for testing purpose on the api website , you will see an 🔒 icon ,  there you can paste your generated token then you can call Certificate api to test whether it is returning the binary data or not.
